app/src/test/test_states/test_track_objects.ts

```diff
@@ -1,3 +1,5 @@
+import { AttributeToolType } from '../../js/common/types'
```

Comment:
better to not use predefined enum or constants in tests. We want to import the raw json object here to make sure we don't break the backward compatibility.